data/nav.yml

```
@@ -224,8 +224,8 @@
           path: "pipelines/artifacts"
         - name: "Pipeline tags"
           path: "pipelines/tags"
-        - name: "Build retention"
-          path: "pipelines/build-retention"
+        - name: "Build export"
```

This is a feature name, which assumes people know what it is. I think we could use a title that uses these key words but speaks more to a user's goal:

```suggestion
        - name: "Export build data"
```

What do you think?

---

Yeah that makes sense 👍